packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,844 @@
+  <p class="dummy-paragraph">It is rectangular (without a radius) and a visible border only at the bottom. Typically
+    only has adjacent whitespace to the bottom of it, meaning it's usually flush to the parent container.
+  </p>
+  <p class="dummy-paragraph">
+    It can have an
+    <strong>icon</strong>
+    (optional), a
+    <strong>title</strong>
+    and/or
+    <strong>description</strong>
+    (required to have at least one of the two), some
+    <strong>actions</strong>
+    (optional) and a
+    <strong>close</strong>
+    button (optional).</p>
+
+  <h4 class="dummy-h5">Inline</h4>
+  <p class="dummy-paragraph">
+    This alert is intended for
+    <strong>system-generated messages</strong>.</p>
+  <p class="dummy-paragraph">
+    It has a border on all sides and a radius. Typically it has adjacent whitespace on all four sides.
+  </p>
+  <p class="dummy-paragraph">
+    It can have an
+    <strong>icon</strong>
+    (optional), a
+    <strong>title</strong>
+    and/or
+    <strong>description</strong>
+    (required to have at least one of the two), some
+    <strong>actions</strong>
+    (optional) and a
+    <strong>close</strong>
+    button (optional).</p>
+
+  <h4 class="dummy-h5">Compact</h4>
+  <p class="dummy-paragraph">
+    This alert is intended for
+    <strong>system-generated messages</strong>.</p>
+  <p class="dummy-paragraph">
+    It's without border or internal padding, and so it has smaller proportions than the others.
+  </p>
+  <p class="dummy-paragraph">
+    It only contains an
+    <strong>icon</strong>
+    and
+    <strong>description</strong>
+    (hence they are both required for this type of alert).</p>
+  <p class="dummy-paragraph">The default icon is also slightly different from the other alert types: it's filled instead
+    of outlined.
+  </p>
+
+  <h4 class="dummy-h5">Toast</h4>
+  <p class="dummy-paragraph">
+    This alert is intended for
+    <strong>user-triggered messages</strong>.</p>
+  <p class="dummy-paragraph">
+    It has a border on all sides, a radius, and elevation (box-shadow). Typically it displays brief, temporary
+    notifications and disappears on its own (after a time interval) or as a result of a user interacting with it.
+  </p>
+  <p class="dummy-paragraph">
+    It can have an
+    <strong>icon</strong>
+    (optional), a
+    <strong>title</strong>
+    and/or
+    <strong>description</strong>
+    (required to have at least one of the two), some
+    <strong>actions</strong>
+    (optional) and a
+    <strong>close</strong>
+    button (optional).</p>
+  <p class="dummy-paragraph">
+    🚨
+    <em><strong>Important</strong>: we provide only the visual styling to this alert, so other features like placement,
+      animations/transitions, and what happens on dismiss, will need to be implemented in your app (eg. with an Ember
+      addon such as
+      <a href="https://github.com/poteto/ember-cli-flash">ember-cli-flash</a>).
```

Comment:
We should not be recommending this addon, since it uses component helpers. See: https://github.com/embroider-build/embroider/blob/main/REPLACING-COMPONENT-HELPER.md

Reply:
Ok, removed.